src/main/src/main/java/org/geoserver/config/util/XStreamPersister.java

```diff
@@ -267,6 +267,12 @@ protected void postEncodeWMTSStore(
 
     Map<Class<?>, String> backwardBreifMap = new HashMap<>();
 
+    /**
+     * List containing classes that should be ignored by XStream. This is required because sometimes generic converter
+     * may be registered in XStream, which will be used instead of dedicated converter from GeoTools
+     */
+    List<Class<?>> backwardBreifIgnored = new ArrayList<>();
```

Comment:
this seems to be a typo
```suggestion
    List<Class<?>> backwardBriefIgnored = new ArrayList<>();
```